composer.json

```diff
@@ -16,11 +16,11 @@
         "psr-4": {"InterNations\\Bundle\\DecoratorBundle\\": "src/"}
     },
     "require": {
-        "symfony/dependency-injection": "~3 | ~4",
-        "symfony/expression-language": "~3 | ~4",
-        "symfony/http-kernel": "~3 | ~4",
-        "symfony/security": "~3 | ~4",
-        "symfony/config": "~3 | ~4",
-        "php": "~7.1"
+        "symfony/dependency-injection": "~3 | ~4 | ~5",
+        "symfony/expression-language": "~3 | ~4 | ~5",
+        "symfony/http-kernel": "~3 | ~4 | ~5",
+        "symfony/security": "~3 | ~4 | ~5",
+        "symfony/config": "~3 | ~4 | ~5",
+        "php": "~7.1 | ~8"
```

Comment:
Should we do more granular requirement for php8 versions ?

Reply:
I think it got automatically updated when running composer update. Symfony 5 is compatible with PHP 7.2.5 and later version. But I am open to specification

Reply:
not compatible with 8